mhkit/wave/io/cdip.py

```diff
@@ -348,8 +354,12 @@ def request_parse_workflow(
 
             # Check the cache for each individual year
             hash_params = f"{station_number}-{parameters}-{start_date}-{end_date}"
-            year_data = handle_caching(hash_params, cache_dir)
-            if year_data[:2] == (None, None):
+            year_data, _, _ = handle_caching(
+                hash_params,
+                cache_dir,
+                cache_content={"data": None, "metadata": None, "write_json": None},
+            )
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args